src/utils/combineReducers.js

```diff
@@ -47,5 +56,13 @@ export default function combineReducers(reducers) {
       );
       return newState;
     });
+
+    warning(
+      verifyStateShape(state, finalState),
+      'Shape of initial state object given to createStore does not ' +
+      'match shape of state object returned by reducer'
```

Comment:
By this point the user does not really define a reducer. All they do is pass `reducers` object (which they might get from `import * as something` ES6 import). Therefore they won't understand which reducer the warning message is referring to.

The warning should instead say something like `Unexpected key "<key>" in the initialState will be ignored. Expected to find one of the known reducer keys instead: "oneReducer", "otherReducer", ...`.